src/server/endpoints/covidcast.py

```diff
@@ -561,6 +561,26 @@ def handle_geo_coverage():
 
     return execute_query(q.query, q.params, fields_string, [], [])
 
+@bp.route("/indicator_geo_coverage", methods=("GET", "POST"))
+def handle_indicator_geo_coverage():
```

Comment:
im not sure how much this actually matters, but do we wanna change the endpoint name to be more similar to the structure of `/geo_coverage`?
```suggestion
@bp.route("/indicator_coverage", methods=("GET", "POST"))
def handle_indicator_coverage():
```
or instead, maybe even rename the other endpoint to "`/geo_indicator_coverage`"?